docs/pull_request_template.md

```diff
@@ -2,6 +2,11 @@
 
 Fixes #
 
+- What work was done?
+- Why was this work done?
+- What feature / user story is it for?
+- Any relevant links or images / screenshots
+
```

Comment:
awesome thanks for updating the fix description, will definitely be helpful for future PRs